modules/users/client/controllers/admin/user.client.controller.js

```diff
@@ -14,7 +14,27 @@
     vm.user = user;
     vm.remove = remove;
     vm.update = update;
+    vm.changeRole = changeRole;
     vm.isContextUserSelf = isContextUserSelf;
+    vm.role = null;
+
+    $scope.roleOptions = [{
+      id: 'ta', //Name will be displayed in the html, id will be used on the database.
+      name: 'TA'
+    },
```

Comment:
is it possible to grab these from the back end? Or, put it in a service? that way we dont have to worry about duplication of code like this.

Reply:
It could be done on the backend instead but I really don't see how that's any better as far as duplication. The model demands it in the lowercase format so we have to convert it at some point. We want capitalized formatting on the frontend.